step/src/main/java/com/google/sps/CalendarManager.java

```diff
@@ -0,0 +1,150 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import java.lang.IllegalStateException;
+import java.time.format.DateTimeFormatter;
+import java.time.ZonedDateTime;
+import java.util.Arrays;
+import java.util.List;
+import java.util.ArrayList;
+import com.google.api.client.auth.oauth2.Credential;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.client.util.DateTime;
+import com.google.api.services.calendar.Calendar;
+import com.google.api.services.calendar.model.Event;
+import com.google.api.services.calendar.model.EventDateTime;
+import com.google.api.services.calendar.model.EventAttendee;
+import com.google.api.services.calendar.CalendarScopes;
+
+// TODO: method to return true/false is user is authenticated (javascript support too)
+// TODO: add javascript support for checking if a user is validated
+// TODO: add test cases
+// TODO: conform to Google Java style
+
+public class CalendarManager {
+  private CalendarManager() {}
+
+  public static void createMatchEvent(String hostUserId, String guestUserId,
+                                      int year, int month, int day, int hour, int minute) {
+    createMatchEvent(new User(hostUserId), new User(guestUserId),
```

Comment:
You might want to check that the year, month, etc values are valid before creating a new match event (unless of course you're doing that on the front-end, in which case, ignore this).

Reply:
Good point, this is one of the things I forgot I needed to do.